azure-pipelines.yml

```diff
@@ -1,20 +1,17 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
 - master
 
 pool:
   vmImage: 'ubuntu-16.04'
 
 # containers must contain curl & bzip2 in order to install Miniconda.
-# Azure has issues with running apt-get inside of contains because of the 
+# Azure has issues with running apt-get inside of contains because of the
 # way it mounts them. Most base images on dockerhub do not contain curl & bzip2
 # see https://github.com/Microsoft/azure-pipelines-agent/issues/2043
 strategy:
   matrix:
+    ubuntu16:
+      containerImage: buildpack-deps:18.04-curl
```

Comment:
This is 18.04 not 16.04. We have this twice now. Also, in case you meant to use buildpack-deps:16.04-curl, that container is missing bzip2 and I couldn't work out how to install it.

Reply:
> This is 18.04 not 16.04.

Ahh!!! This is why PRs are important :wink: 

I'll looks into the bzip2 issue